### ISSUE TITLE
Missing files on `code-examples/tutorials/dbi`

### ISSUE THREAD

**Issue by renatomello** [qiboteam/qibo#1228]
Dozens of warnings are raised when compiling the documentation in `master` due to missing files in the path mentioned in the title

```python
WARNING: toctree contains reference to nonexisting document 'code-examples/tutorials/dbi/dbi'
```

**Comment by scarrazza** [comment]
@BrunoLiegiBastonLiegi, can we close this issue?

**Comment by BrunoLiegiBastonLiegi** [comment]
I still see one warning about that file, but several others are raised. Furthermore, the dbi tutorial make the building of the docs quite slow.

```
/home/andrea/git/qibo/doc/source/code-examples/tutorials/dbi/dbi_group_commutator_tests.rst: WARNING: document not readable. Ignored.

/home/andrea/git/qibo/doc/source/code-examples/tutorials/dbi/dbi_group_commutator_tests.rst: WARNING: document not readable. Ignored.
/home/andrea/git/qibo/src/qibo/models/error_mitigation.py:docstring of qibo.models.error_mitigation.apply_randomized_readout_mitigation:38: ERROR: Unexpected indentation.
/home/andrea/git/qibo/src/qibo/models/error_mitigation.py:docstring of qibo.models.error_mitigation.apply_randomized_readout_mitigation:1: WARNING: duplicate object description of qibo.models.error_mitigation.apply_randomized_readout_mitigation, other instance in api-reference/qibo, use :noindex: for one of them
/home/andrea/git/qibo/src/qibo/models/error_mitigation.py:docstring of qibo.models.error_mitigation.apply_randomized_readout_mitigation:38: ERROR: Unexpected indentation.
/home/andrea/git/qibo/src/qibo/gates/abstract.py:docstring of qibo.gates.abstract.Gate.target_qubits:1: WARNING: duplicate object description of qibo.gates.abstract.Gate.target_qubits, other instance in api-reference/qibo, use :noindex: for one of them
/home/andrea/git/qibo/src/qibo/gates/abstract.py:docstring of qibo.gates.abstract.Gate.control_qubits:1: WARNING: duplicate object description of qibo.gates.abstract.Gate.control_qubits, other instance in api-reference/qibo, use :noindex: for one of them
/home/andrea/git/qibo/src/qibo/result.py:docstring of qibo.result.MeasurementOutcomes.frequencies:13: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/home/andrea/git/qibo/src/qibo/quantum_info/clifford.py:docstring of qibo.quantum_info.clifford.Clifford.frequencies:15: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/home/andrea/git/qibo/src/qibo/quantum_info/utils.py:docstring of qibo.quantum_info.utils.hellinger_fidelity:1: WARNING: duplicate object description of qibo.quantum_info.hellinger_fidelity, other instance in api-reference/qibo, use :noindex: for one of them
/home/andrea/git/qibo/src/qibo/backends/abstract.py:docstring of qibo.backends.abstract.Backend.execute_circuits:1: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/home/andrea/git/qibo/src/qibo/backends/clifford.py:docstring of qibo.backends.clifford.CliffordBackend.zero_state:1: WARNING: Inline substitution_reference start-string without end-string.
/home/andrea/git/qibo/doc/source/code-examples/applications.rst:157: WARNING: toctree contains reference to nonexisting document 'code-examples/tutorials/dbi/dbi'
/home/andrea/git/qibo/doc/source/code-examples/tutorials/dbi/dbi_strategy_Ising_model.ipynb:511: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/home/andrea/git/qibo/doc/source/getting-started/installation.rst:263: ERROR: Document may not end with a transition.

/home/andrea/git/qibo/doc/source/code-examples/tutorials/quantum_networks/README.md: WARNING: document isn't included in any toctree
```

**Comment by scarrazza** [comment]
@MatteoRobbiati could you please double check this?

**Comment by renatomello** [comment]
Closed by #1402
